test/js/search/search-query-display.test.js

```diff
@@ -1,61 +1,94 @@
-import React from 'react';
-import * as ReactAll from 'react';
-import { mount } from 'enzyme';
+import React from 'react'
+import { mount } from 'enzyme'
 
-const fetch = require('node-fetch');
+const fetch = require('node-fetch')
+
+import SearchQueryDisplay, { ClearAllButton } from 'components/SearchQueryDisplay'
+import { PropsStudySearchProvider } from 'components/search/StudySearchProvider'
+import KeywordSearch from 'components/KeywordSearch'
+import FacetControl from 'components/FacetControl'
 
-import SearchQueryDisplay from 'components/SearchQueryDisplay';
 
 const oneStringFacet = [
-  {id: 'species', filters: [{id: "NCBITaxon_9606", name: "Homo sapiens"}]}
+  { id: 'species', filters: [{ id: 'NCBITaxon_9606', name: 'Homo sapiens' }] }
 ]
 
 const twoStringFacets = [
-  {id: 'disease', filters: [{id: "id1", name: "disease1"}]},
-  {id: 'species', filters: [{id: "NCBITaxon_9606", name: "Homo sapiens"}]}
+  { id: 'disease', filters: [{ id: 'id1', name: 'disease1' }] },
+  { id: 'species', filters: [{ id: 'NCBITaxon_9606', name: 'Homo sapiens' }] }
 ]
 
 const stringAndNumericFacets = [
-  {id: 'species', filters: [
-    {id: "NCBITaxon_9606", name: "Homo sapiens"},
-    {id: "NCBITaxon_10090", name: "Mus musculus"}
-  ]},
-  {id: "organism_age", filters: {min: 14, max: 180, unit: "years"}}
+  {
+    id: 'species', filters: [
+      { id: 'NCBITaxon_9606', name: 'Homo sapiens' },
+      { id: 'NCBITaxon_10090', name: 'Mus musculus' }
+    ]
+  },
+  { id: 'organism_age', filters: { min: 14, max: 180, unit: 'years' } }
 ]
 
 describe('Search query display text', () => {
   it('renders a single facet', async () => {
     const wrapper = mount((
       <SearchQueryDisplay facets={oneStringFacet} terms={''}/>
     ))
-    expect(wrapper.text().trim()).toEqual(': Metadata contains (species: Homo sapiens)')
+    expect(wrapper.text().trim()).toEqual(': Metadata contains (species: Homo sapiens) Clear All')
```

Comment:
see above comment about finding the query-text span so you don't have to include 'Clear All' in this test